drivers/include/mpu9150.h

```diff
@@ -0,0 +1,340 @@
+/*
+ * Copyright (C) 2015 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    mpu9150 MPU-9150
+ * @ingroup     drivers
+ * @brief       Device driver interface for the MPU-9150
+ * @{
+ *
+ * @file
+ * @brief       Device driver interface for the MPU-9150
+ *
+ * @author      Fabian Nack <nack@inf.fu-berlin.de>
+ */
+
+#ifndef MPU9150_H_
+#define MPU9150_H_
+
+#include "periph/i2c.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name Sample rate macro definitions
+ * @{
+ */
+#define MPU9150_MIN_SAMPLE_RATE     (4)
+#define MPU9150_MAX_SAMPLE_RATE     (1000)
+#define MPU9150_DEFAULT_SAMPLE_RATE (50)
+#define MPU9150_MIN_COMP_SMPL_RATE  (1)
+#define MPU9150_MAX_COMP_SMPL_RATE  (100)
+/** @} */
+
+/**
+ * @name Power Management 1 register macros
+ * @{
+ */
+#define MPU9150_PWR_WAKEUP          (0x00)
+#define MPU9150_PWR_PLL             (0x01)
+#define MPU9150_PWR_RESET           (0x80)
+/** @} */
+
+/**
+ * @name Power Management 2 register macros
+ * @{
+ */
+#define MPU9150_PWR_GYRO            (0x07)
+#define MPU9150_PWR_ACCEL           (0x38)
+/** @} */
+
+/**
+ * @name Sleep times in microseconds
+ * @{
+ */
+#define MPU9150_COMP_MODE_SLEEP_US  (1000)
+#define MPU9150_BYPASS_SLEEP_US     (3000)
+#define MPU9150_PWR_CHANGE_SLEEP_US (50000)
+#define MPU9150_RESET_SLEEP_US      (100000)
+/** @} */
+
+/**
+ * @name MPU-9150 compass operating modes and reg values
+ * @{
+ */
+#define MPU9150_COMP_POWER_DOWN     (0x00)
+#define MPU9150_COMP_SINGLE_MEASURE (0x01)
+#define MPU9150_COMP_SELF_TEST      (0x08)
+#define MPU9150_COMP_FUSE_ROM       (0x0F)
+#define MPU9150_COMP_WHOAMI_ANSWER  (0x48)
+/** @} */
+
+/**
+ * @brief Power enum values
+ */
+typedef enum {
+    PWR_OFF = 0x00,
+    PWR_ON = 0x01,
```

Comment:
As these are public enum values, they should be prepended by `MPU9150_*`. Same for all following enum values in this file...